### PR TITLE
chore: cli docs generate after release

### PR DESCRIPTION
**Pull request by bobvandenhoogen**

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -51,3 +51,19 @@ jobs:
...
+      - name: Update CLI docs
+        env:
+          GITHUB_TOKEN: ${{secrets.GH_TOKEN}}
+          BOT_USERNAME: ${{secrets.BOT_USERNAME}}
+          BOT_EMAIL: ${{secrets.BOT_EMAIL}}
+        run: |
+          make docs
+          git config --global user.email "${BOT_EMAIL}"
+          git config --global user.name "${BOT_USERNAME}"
+          git clone "https://git:${GITHUB_TOKEN}@github.com/getstrm/pace.git"
```
Comment (trietsch) [inline .github/workflows/build.yaml:63]: Maybe an explicit branch checkout would be wise here (i.e. `alpha`).

File: scripts/update_summary.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+text = ""
+with open('./pace/docs/SUMMARY.md', 'r') as file:
+  text = file.read()
+with open('./pace/docs/SUMMARY.md', 'w') as file:
+  replacement = ""
+  for fn in sorted([f[len('./generated_docs/'):] for f in glob('./generated_docs/pace*')]):
+    filename = fn.replace('_', "\\_")
+    replacement+=f"""{'  '*len(fn.split('_'))}* [{fn.split('_')[-1].replace('.md','')}](cli-docs/{filename})\n"""
+  out = re.sub('([\s\S]*\[CLI Reference\]\(reference/cli-reference/README.md\))[\S\s]*?\n(\*[\s\S]*)', f'\\1\n{replacement}\\2', text)
```
Comment (trietsch) [inline scripts/update_summary.py:12]: I presume this has been tested in trial and error fashion :)? I can't really review this, but it either works or it doesn't.

**Review by trietsch — CHANGES_REQUESTED** [review]

File: scripts/update_summary.py (hunk 2 above)
Comment (bobvandenhoogen) [reply scripts/update_summary.py:12]: Indeed it did work in trial and error

File: .github/workflows/build.yaml (hunk 1 above)
Comment (bobvandenhoogen) [reply .github/workflows/build.yaml:63]: done

**Review by trietsch — APPROVED** [review]

**Pull request merged**